spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/TaskOperations.java

```diff
@@ -114,6 +115,15 @@
 	 */
 	PagedModel<TaskExecutionResource> executionListByTaskName(String taskName);
 
+	/**
+	 * List task executions known to the system filtered by task name.
+	 *
+	 * @param taskName of the executions.
+	 * @param pageable page request.
+	 * @return the paged list of task executions for the given task name
+	 */
+	PagedModel<TaskExecutionResource> executionListByTaskName(String taskName, Pageable pageable);
```

Comment:
I wonder why you needed to added `Pageable` as an explicit argument as it should be part of the REST request itself. The REST controller [TaskExecutionController](https://github.com/spring-cloud/spring-cloud-dataflow/blob/master/spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/TaskExecutionController.java#L142) takes of this. You can also refer to the REST [guide](https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#api-guide-resources-task-executions-list-by-name) for the same.

Reply:
@ilayaperumalg From what I understood in the issue, it required for a function to be created which takes in page size and number for getting executionListByTaskName. Tried to follow what @cppwfs had mentioned. Maybe I was unable to understand the issue correctly.

Reply:
ok, let's wait for @cppwfs to confirm if we really need this new method (based on the other PR changes).

Reply:
Hello @ilayaperumalg it was a part of the original story.   I've attached that story to this PR.

Reply:
Hey @ilayaperumalg @cppwfs What should we do about this PR then? Do we need this change?

Reply:
@siddhantsorann Given the original PR got closed, let's close this for now. Please feel free to re-open this if you require this for any other changes. Thanks again!